singulostation/code/game/machinery/autodoc.dm

```diff
@@ -161,4 +161,7 @@
 	if(panel_open)
 		overlays += "[icon_state]_panel"
 
+/obj/machinery/autodoc/on_emag(mob/user)
+	..()
+
```

Comment:
Isn't it redundant though? Or do you need to prevent it from returning?

Reply:
Ah, I see, the empty line with no indentation confused me, carry on

Reply:
It needs to call the parent, and this is how it's done everywhere else I checked
See: #223

Reply:
Though, now that I think about it, should_emag should probably return true

Reply:
Ah, looks like I need to se `emag_toggleable` to true